test/validate/mcoe_test.py

```diff
@@ -109,7 +109,7 @@ def test_no_null_rows_mcoe(pudl_out_mcoe, live_dbs, df_name, thresh):
         ("hr_by_gen", 557_819, 46_535),
         ("fuel_cost", 557_819, 46_535),
         ("capacity_factor", 604_594, 50_448),
-        ("mcoe", 1_077_537, 523_343),
+        ("mcoe", 604_642, 50_448),
```

Comment:
The numbers prior to [adding the 2021 data](https://github.com/catalyst-cooperative/pudl/pull/2017/files) were `(599_722, 50_038)`. These increases are more on par with the increases for the other analysis tables. These row counts align with the recent nightly build outputs.